Engine/src/main/java/dk/gruppeseks/bodtrd/engine/Game.java

```diff
@@ -39,7 +41,7 @@
     private final GameData _gameData = new GameData();
     private List<IEntityProcessor> _entityProcessors = new ArrayList<>();
     private Map<Integer, Entity> _world = new ConcurrentHashMap<>();
-    private List<GamePluginSPI> _gamePlugins;
+    private Set<GamePluginSPI> _gamePlugins;
```

Comment:
i wonder if we should override hashCode and equals in the game plugin implementations. or if this will detect any changes correctly